### PR TITLE
Allow to disable comment loading/rendering on group_overview

### PR DESCRIPTION
**Pull request by okurz**
Save some time and performance when not needed.

### CODE REVIEW

File: lib/OpenQA/WebAPI/Controller/Main.pm [hunk 1]
```diff
@@ -87,14 +87,15 @@ sub group_overview {
     $time_limit_days = 0 unless looks_like_number($time_limit_days);
 
     $self->app->log->debug("Retrieving results for up to $limit_builds builds up to $time_limit_days days old");
-    my $only_tagged = $self->param('only_tagged') // 0;
-    my $group = $self->db->resultset($resultset)->find($self->param('groupid'));
+    my $only_tagged      = $self->param('only_tagged') // 0;
+    my $comments_enabled = $self->param('comments')    // 1;
```
Comment (coolo) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: I don't like this option - especially as it pollutes the template too

Comment (okurz) [inline lib/OpenQA/WebAPI/Controller/Main.pm]: we could just overwrite the query param itself if you prefer

**Comment by coolo** [comment]
how about limiting the comments loaded to let's say 14 days and make that configurable and then you abuse that setting. But do it properly and visible instead of some hidden option for someone's office entertainment.

File: templates/main/group_overview.html.ep [hunk 2]
```diff
@@ -39,16 +39,18 @@
 %= include 'main/group_builds', build_results => $build_results, group => $group, children => undef, default_expanded => 1
 %= include 'main/more_builds', limit_builds => $limit_builds
 
-<h2>Comments</h2>
-% for my $comment (reverse @$comments) {
-    %= include 'comments/comment_row', comment_id => $comment->id, comment => $comment, user => $comment->user, context => {type => 'group', id => $group->{id}}, put_action => 'apiv1_put_group_comment', delete_action => 'apiv1_delete_group_comment'
-% }
+% if ($comments_enabled) {
```
Comment (kraih) [inline templates/main/group_overview.html.ep]: Couldn't this just be `% if (@$comments) {`? Then only the controller code has to deal with comment limits.

**Comment by okurz** [comment]
I would provide some benefit already as is. But if you won't accept it as is then I can't get it done ATM.

**Pull request closed without merging**